.github/workflows/ci.yml

```diff
@@ -90,6 +90,11 @@ jobs:
               body: "Visual regression tests failed 😢. You can download the failure diffs from the 'Artifacts' section of the failed CI run. To update the tests, run `npm run jest:visual-tests:update` locally."
             })
 
+      - name: Percy Test
```

Comment:
We will probably also want this to not run on dependabot PRs and not forks - to avoid busting limits and prevent failures so will need to add under here:

```
if: github.repository == github.event.pull_request.head.repo.full_name && github.actor != 'dependabot[bot]'
```